cmd/vulcan-host-discovery/main.go

```diff
@@ -87,13 +89,13 @@ func discoveredHosts(targetSubnet string, knownHosts []string, nmapReport *gonma
 	 * summary to "Discovered Hosts" instead of "Unknown Hosts".
 	 */
 	if len(knownHosts) == 0 {
-		exposedVuln.Summary = "Discovered Hosts"
-		exposedVuln.Description = "At least one host has been discovered in the network"
+		exposedVuln.Summary = "Discovered Host"
+		exposedVuln.Description = "A host has been discovered in the network"
 		exposedVuln.Recommendations = []string{
-			"Identify that all of the discovered hosts are known hosts",
+			"Identify if the discovered host is a known host",
 		}
 		exposedVuln.Score = report.SeverityThresholdNone
-		exposedVuln.ImpactDetails = "If any of the discovered hosts is not known, it may be an unauthorized malicious host."
+		exposedVuln.ImpactDetails = "If the discovered host is not known, it may be an unauthorized malicious host."
 	}
 
 	add := false
```

Comment:
Due to the changes in the logic you applied, I think we won't need to use this anymore.

Also we could replace the content on [lines 137-138](https://github.com/adevinta/vulcan-checks/pull/154/files#diff-29023f4b1137a4edd3c3a579d05cd02401b38499937ba0e09f6329f5d76e2e34R137-R138) by:
```
vuln := exposedVuln
```
And from there make references to that `vuln` instead. Otherwise at the end of the loop all your entries in the array might end having the same values.